src/styles/globals.css

```diff
@@ -8,101 +8,151 @@
   --background-end-rgb: 255, 255, 255;
 }
 
+html,
+body {
+  width: 100%;
+  height: 100vh;
 
-@font-face {
-  font-family: 'system1';
-  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
-  font-size: 24px;
-  font-style: normal;
-}
-@font-face {
-  font-family: 'system2';
-  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
-  font-size: 20px;
-  font-style: normal;
-}
-@font-face {
-  font-family: 'system3';
-  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
-  font-size: 18px;
-  font-style: normal;
-}
-@font-face {
-  font-family: 'system4';
-  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
-  font-size: 16px;
-  font-style: normal;
-}
-@font-face {
-  font-family: 'system5';
-  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
-  font-size: 14px;
-  font-style: normal;
-}
-@font-face {
-  font-family: 'system6';
-  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff') format('woff');
-  font-size: 12px;
-  font-style: normal;
-}
-
-
+  padding: 0;
+  margin: 0;
+  position: fixed;
+  overflow: hidden;
 
-
-@font-face {
-  font-family: 'system4_medium';
-  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Medium.woff') format('woff');
-  font-size: 16px;
-  font-style: normal;
-}
-@font-face {
-  font-family: 'system5_medium';
-  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Medium.woff') format('woff');
-  font-size: 14px;
-  font-style: normal;
+  /* iOS only */
+  @supports (-webkit-touch-callout: none) {
+    height: -webkit-fill-available;
+  }
 }
-@font-face {
-  font-family: 'system6_medium';
-  src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Medium.woff') format('woff');
-  font-size: 12px;
-  font-style: normal;
+
+* {
+  box-sizing: border-box !important;
+  -moz-box-sizing: border-box !important; /* Firefox */
+  -webkit-box-sizing: border-box !important; /* Safari */
+  /*아이폰 버튼 클릭 깜박임 해제*/
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
 }
 
+@layer base {
+  @font-face {
+    font-family: 'system1';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Regular.woff')
+      format('woff');
+    font-size: 24px;
+    font-style: normal;
+    line-height: 30px;
+  }
```

Comment:
layer base라는 걸 추가했어요. 이게 뭐냐면
[공식문서 참고](https://v1.tailwindcss.com/docs/adding-base-styles)
By using the @layer directive, Tailwind will automatically move those styles to the same place as @tailwind base to avoid unintended specificity issues.
(@layer 지시어를 사용하여 Tailwind는 의도하지 않은 특수성 문제를 방지하기 위해 해당 스타일을 @tailwind base와 동일한 위치로 자동으로 이동합니다.)

라고 하더라구요. 공식문서에서 권장하는 방식인 것 같아 추가했습니다.